docs/default_cni_plugin.md

```diff
@@ -93,14 +93,50 @@ When `disableOVSInit` is set to true in ovn-kubernetes CNI config file, OVS brid
 ```bash
 sudo systemctl enable openvswitch --now
 sudo ovs-vsctl add-br br-ex
-sudo ovs-vsctl add-port br-ex <physical-interface-name>
+sudo ovs-vsctl add-port br-ex enp1s0
+sudo ip link set enp1s0 up
 sudo ip link set br-ex up
 ```
 
-Replace `<physical-interface-name>` with the network interface name where node IP address is assigned to.
-Once br-ex up is up, assign the node IP address to br-ex bridge.
+Replace `enps10` with the network interface name in your own environment where node IP address is assigned to.<br>
+Once br-ex is up, assign the node IP address to br-ex bridge.<br>
 
-> Adding physical interface to br-ex bridge will disconnect ssh connection on node IP address.
+**NOTE:* Adding physical interface `enp1s0` to br-ex bridge will disconnect ssh connection on node IP address.
```

Comment:
I would suggest we leave this part out of the PR.

If we give instructions to do this, we should probably do things via the network manager to make sure the changes are persistent across reboots.

WDYT?

Reply:
Agree, removed from this PR.